netweaver/setup/swap_space.sls

```diff
@@ -0,0 +1,37 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% for node in netweaver.nodes if host == node.host %}
+
+{% set swap_file = '/var/lib/swap/swapfile' %}
+
+create_swap_dir:
```

Comment:
We should use the `file.directory` state to make this idempotent instead of using `cmd.run`